CRM/Civicase/Hook/BuildForm/TokenTree.php

```diff
@@ -0,0 +1,392 @@
+<?php
+
+/**
+ * Attaches a new token tree to the form.
+ */
+class CRM_Civicase_Hook_BuildForm_TokenTree {
```

Comment:
@shahrukh-compucorp , Can we also add tests for this class? We should plan to add tests for new changes as much as possible even if it is basic tests.

Reply:
I will create a new ticket for adding tests for this class and add it to the board